src/clients/index.ts

```diff
@@ -0,0 +1,2 @@
+export * as AcrossConfigStoreClient from "./AcrossConfigStoreClient";
```

Comment:
I would not export everything as `AcrossConfigStoreClient`, otherwise you'll have to instantiate as a new client via `AcrossConfigStoreClient.AcrossConfigStoreClient`, right?

Maybe just `export { GLOBAL_CONFIG_STORE_KEYS, AcrossConfigStoreClient } from "./AcrossConfigStoreClient"`

Reply:
Good call - I've updated it in a later commit below